primefaces-showcase/src/main/webapp/WEB-INF/template.xhtml

```diff
@@ -10,13 +10,15 @@
 
     <h:head>
         <f:facet name="first">
-            <script async="async" src="https://www.googletagmanager.com/gtag/js?id=UA-93461466-1"></script>
-            <script>
-                window.dataLayer = window.dataLayer || [];
-                function gtag(){dataLayer.push(arguments);}
-                gtag('js', new Date());
-                gtag('config', 'UA-93461466-1');
-            </script>
+            <ui:fragment rendered="#{request.serverName != 'localhost'}">
```

Comment:
I added this rendered so the Google Tag Manager is never loaded when running localhost showcase.

Reply:
Isn't `#{facesContext.application.projectStage ne 'Development'}"` better?

Reply:
Actually we should only render it on production

Reply:
I think the problem is we have the web.xml hardcoded to Production mode that is why I didn't do it.

Reply:
And this was only for the showcase for us developers so i thought checking localhost was OK.

Reply:
I have no idea I know they just run Jetty but not sure if they run it clustered or standalone.